src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/M2ArgumentCollection.java

```diff
@@ -13,6 +13,7 @@
 import java.util.List;
 
 public class M2ArgumentCollection extends AssemblyBasedCallerArgumentCollection {
+    private static final long serialVersionUID = 9341L;
```

Comment:
What's serial version UID for? (I will leave a bunch of comments/questions on github - maybe we can go through them in person on Tuesday?)